### ISSUE TITLE
Training Loss

### ISSUE THREAD

**Issue by gc0127** [yeguixin/captcha_solver#23]
Hi  

I have used the same parameters to train the LeNet model, but it is giving  very high training loss and the loss keeps on increasing exponentially with the first epoch itself. Any suggestions for such a behavior.

Thanks in advance.
